components/post_view/post_list/post_list_virtualized.jsx

```diff
@@ -168,14 +168,11 @@ export default class PostList extends React.PureComponent {
 
     getSnapshotBeforeUpdate(prevProps) {
         if (this.postListRef && this.postListRef.current) {
-            const postsAddedAtTop = this.props.postListIds && this.props.postListIds.length !== prevProps.postListIds.length && this.props.postListIds[0] === prevProps.postListIds[0];
+            const postsAddedAtTop = this.props.postListIds && (this.props.postListIds.length !== prevProps.postListIds.length) && (this.props.postListIds[0] === prevProps.postListIds[0]);
             const channelHeaderAdded = this.props.olderPosts.allLoaded !== prevProps.olderPosts.allLoaded;
-            if ((postsAddedAtTop || channelHeaderAdded) && !this.state.atBottom) {
-                const previousScrollTop = this.postListRef.current.scrollTop;
+            if ((postsAddedAtTop || channelHeaderAdded) && !this.state.atBottom && !this.correctionInProgress) {
```

Comment:
Where is `this.correctionInProgress` supposed to be coming from? I can't find any other reference to it